WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -268,6 +276,19 @@ private void deserializeDotComRestResponse(SiteModel site, JSONObject response)
             mRemoteSettings.sortCommentsBy = DESCENDING_SORT;
         }
 
+        JSONObject jetpackProtectWhitelist = settingsObject.optJSONObject(JP_PROTECT_WHITELIST_KEY);
+        if (jetpackProtectWhitelist != null) {
+            JSONArray whitelistItems = jetpackProtectWhitelist.optJSONArray("local");
+            if (whitelistItems != null) {
+                for (int i = 0; i < whitelistItems.length(); ++i) {
+                    String item = whitelistItems.optString(i, "");
+                    if (!item.isEmpty() && !mRemoteJpSettings.jetpackProtectWhitelist.contains(item)) {
```

Comment:
If we are going to have this check, wouldn't it be better to just use a Set? It'll also fix the issue I mentioned about comparing 2 lists.